INSTALL.md

```diff
@@ -121,12 +129,17 @@ Navigate to the cloned directory and ensure your conda environment is active:
 cd qiita
 source activate qiita
 ```
-
+Before we install Qiita, make sure you have a few other tools installed:
+```bash
+sudo apt install gcc
+sudo apt-get install libpq-dev
```

Comment:
@aliu104 This is also specific to Ubuntu. I would suggest adding some text along the lines of - Make sure development libraries and include files for PostgreSQL are available by installing libpq-dev for Ubuntu or similar for other distributions if needed.

Reply:
@aliu104 Ah, it just occurred to me that Ubuntu is likely the core of WSL. In that case, these instructions work for WSL. I believe however that they also apply to installing on Ubuntu as well. If you can note that in the documentation it would be much appreciated.